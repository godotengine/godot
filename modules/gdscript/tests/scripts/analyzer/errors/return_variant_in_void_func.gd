func test() -> void:
	var a
	a = 1
	return a
