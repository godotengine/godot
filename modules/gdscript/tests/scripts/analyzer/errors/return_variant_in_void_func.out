GDTEST_ANALYZER_ERROR
>> ERROR at line 4: A void function cannot return a value.
